### PR TITLE
TxBuilder: Simplify/cleanup some leftovers from past changes

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

File: source/agora/utils/TxBuilder.d [hunk 1]
```diff
@@ -236,9 +236,7 @@ public struct TxBuilder
                 "Address not found in Well-Known keypairs: "
                 ~ out_ref.output.address.toString());
 
-        import Schnorr = agora.crypto.Schnorr;
-        Pair pair = Pair(ownerKP.secret, ownerKP.secret.toPoint());
-        auto sig = Schnorr.sign(pair.v, tx);
+        auto sig = ownerKP.sign(pair.v, tx);
```
Comment (linked0) [inline source/agora/utils/TxBuilder.d]: `pair.v` should be removed.

Comment (kchulj) [reply source/agora/utils/TxBuilder.d]: Yes. Can we call directly from `ownerKP` ?

Comment (Geod24) [reply source/agora/utils/TxBuilder.d]: Oops, fixed

Comment (linked0) [reply source/agora/utils/TxBuilder.d]: > Yes. Can we call directly from `ownerKP` ?

Yes, we can.
https://github.com/bosagora/agora/blob/39c038debbdbfd3052bb7756006a3e6d9a48dc09/source/agora/crypto/Key.d#L92-L96

**Review by kchulj — APPROVED** [review]
LGTM

**Review by linked0 — APPROVED** [review]

**Pull request merged**